src/simtools/schemas/model_parameters/dsum_shaping.schema.yml

```diff
@@ -30,7 +30,6 @@ instrument:
   type:
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for FlashCams?

Reply:
Same, see above.